corehq/apps/reports/generic.py

```diff
@@ -695,6 +695,14 @@ def show_in_navigation(cls, domain=None, project=None, user=None):
         return True
 
     @classmethod
+    def show_in_webuser_permissions(cls, domain=None, project=None, user=None):
```

Comment:
Does this show in that modal when you create or edit a "Role"? If so, these are more than just webuser permissions, as they can also be used for mobile workers (confusing, because it is on the same page as webusers).
I'd rename this to `show_in_user_roles` or something similar.